cli/utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from "fs";
+import * as protos from "@dataform/protos";
+
+export enum WarehouseTypes {
```

Comment:
Can we put this somewhere else, as we are probably going to use it elsewhere? I imagine in core/adapters/index.js is probably a good place to put it